src/components/SiteHeader.vue

```diff
@@ -1,19 +1,67 @@
 <template>
-  <header class="py-4 py-sm-5">
-    <b-navbar class="p-0" toggleable="lg" type="light">
+  <header class="py-2 sticky-top header">
+    <b-navbar class="p-1" toggleable="lg" type="light" id="nav-header">
       <b-navbar-brand>
         <b-navbar-brand class="font-family-headings" to="/">
-          <span class="h3 text-dark">hack</span>
+          <img src="@/assets/images/hack-logo.png" alt="hack-logo" height="80" />
         </b-navbar-brand>
       </b-navbar-brand>
       <b-navbar-toggle target="nav-collapse"></b-navbar-toggle>
       <b-collapse id="nav-collapse" is-nav>
-        <b-navbar-nav class="ml-auto ">
-          <b-nav-item to="/" exact-active-class="active">Home</b-nav-item>
-          <b-nav-item to="/about" active-class="active">About</b-nav-item>
+        <b-navbar-nav class="ml-auto align-middle">
+          <b-nav-item to="/" exact-active-class="active" class="text-primary">Home</b-nav-item>
+          <b-nav-item-dropdown text="Services" class="dropper" right>
+            <b-dropdown-item href="#">Blockchain Development</b-dropdown-item>
+            <b-dropdown-item href="#">Smart Contracts Development</b-dropdown-item>
+            <b-dropdown-item href="#">Smart Contract Audits</b-dropdown-item>
+            <b-dropdown-item href="#">Consulting</b-dropdown-item>
+            <b-dropdown-item href="#">Decentralized application (dApp) development</b-dropdown-item>
+            <b-dropdown-divider></b-dropdown-divider>
+            <b-dropdown-item href="#">All services</b-dropdown-item>
+          </b-nav-item-dropdown>
+          <b-nav-item to="/about" active-class="active" class="text-primary">About us</b-nav-item>
           <b-nav-item to="/blog" active-class="active">Blog</b-nav-item>
+          <b-nav-item to="/contact" active-class="active">
+            <span class="contact">Get in touch</span>
+          </b-nav-item>
         </b-navbar-nav>
       </b-collapse>
     </b-navbar>
   </header>
 </template>
+
+<script>
+export default {
+  created() {
+    window.addEventListener("scroll", this.handleScroll);
+  },
+  destroyed() {
+    window.removeEventListener("scroll", this.handleScroll);
+  },
```

Comment:
```suggestion
   mounted() {
    window.addEventListener('scroll', () => this.handleScroll);
  },
  destroyed() {
    window.removeEventListener('scroll', () => this.handleScroll);
  },
```

The build was failing because `gridsome` is using server-side rendering and `window` is not defined at that time. It is only available on the client-side. So the solution of this is to use other Vue lifecycle hooks which are fired on the client-side - where `window` is already available. (ref: https://github.com/gridsome/gridsome/issues/800#issuecomment-565809662 , https://vuejs.org/v2/api/#Options-Lifecycle-Hooks)

Also using an arrow function will keep `this` unchaged, otherwise it changes the scope - like so:

```
window.addEventListener('scroll', () => this.handleScroll);
```